password_security/models/res_users.py

```diff
@@ -144,6 +144,8 @@ def _validate_pass_reset(self):
             if pass_min <= 0:
                 continue
             write_date = user.password_write_date
+            if not write_date:
+                continue
             delta = timedelta(hours=pass_min)
             if write_date + delta > datetime.now():
```

Comment:
IMHO it could be only in one line since that the variable `delta` is used only one time

But it is only a nitpick related to style not change if you disagree

```suggestion
            if write_date and write_date + timedelta(hours=pass_min) > datetime.now():
```